### ISSUE TITLE
The PyTorch model won't load in PyTorch 0.4.*

### ISSUE THREAD

**Issue by l3robot** [junyanz/interactive-deep-colorization#54]
Hi, 
It is quite a small details, though useful to know. The pre-trained model won't work in PyTorch 0.4.* because normalization layers now track the number of batches with the tensor `num_batches_tracked` [see [code](https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html#BatchNorm2d)]. When loading a `state_dict`, it then expects these tensors and crashes on a `KeyError`. Maybe just adding the pytorch version in requirements could help future users.
Thanks

**Comment by junyanz** [comment]
Cool. I updated the README and add PyTorch 0.5. (I added a patch to the network loading code just in case people are still using PyTorch 0.4. )

**Comment by l3robot** [comment]
Excellent! thanks 
sry for the delay
